Makefile

```diff
@@ -144,7 +144,7 @@ $(PACKAGES_BUILD_DIR):
 .rpm: $(PACKAGES_BUILD_DIR)
 	fpm -p $(PACKAGES_BUILD_DIR) -t rpm $(FPM_OPTS) $(FPM_FILES)
 .apk: $(PACKAGES_BUILD_DIR)
-	fpm -p $(PACKAGES_BUILD_DIR) -t apk $(FPM_OPTS) --depends=libc6-compat --depends=bash --depends=libexecinfo $(FPM_FILES)
+	fpm -p $(PACKAGES_BUILD_DIR) -t apk $(FPM_OPTS) --depends=libc6-compat --depends=bash --depends=curl --depends=libexecinfo $(FPM_FILES)
```

Comment:
I added `curl` as a dependency but I have mixed feelings about it, for the following reasons:
1. if users installed php from a package 99.99% it is build `--with-curl` => libcurl is installed
2. if the customer built php in the machine and has `--with-curl` => libcurl-dev is installed
3. if the customer built/installed php without curl, we would install the dependency but we cannot work in any case, so I might leave it here as a "reference" is anyone inspect the package, but I believe is pretty much useless.

Reply:
For #3: in the future I think all curl communication will be done at the C level, so while today they probably will not work without PHP's curl, in the future it will.

Reply:
I think that this alone is a very good selling point. Thanks for pointing that out.